### PR TITLE
Update ImagePullPolicy to "IfNotPresent"

### PR DESCRIPTION
**Pull request by alexc2684**
#530 

Signed-off-by: Alex Chan <alexc2684@gmail.com>

### CODE REVIEW

**Comment by philkuz** [comment]
You'll want to also add this change to the following files:
```
❯ git ls-files k8s/  | grep -E 'statefulset|deployment|daemonset'                                                                                                                
k8s/cloud/base/api_deployment.yaml
k8s/cloud/base/artifact_tracker_deployment.yaml
k8s/cloud/base/auth_deployment.yaml
k8s/cloud/base/config_manager_deployment.yaml
k8s/cloud/base/cron_script_deployment.yaml
k8s/cloud/base/indexer_deployment.yaml
k8s/cloud/base/metrics_deployment.yaml
k8s/cloud/base/ory_auth/hydra/hydra_deployment.yaml
k8s/cloud/base/ory_auth/kratos/kratos_deployment.yaml
k8s/cloud/base/plugin_deployment.yaml
k8s/cloud/base/profile_deployment.yaml
k8s/cloud/base/project_manager_deployment.yaml
k8s/cloud/base/proxy_deployment.yaml
k8s/cloud/base/scriptmgr_deployment.yaml
k8s/cloud/base/vzconn_deployment.yaml
k8s/cloud/base/vzmgr_deployment.yaml
k8s/cloud/prod/api_deployment.yaml
k8s/cloud_deps/base/nats/statefulset.yaml
k8s/cloud_deps/base/stan/statefulset.yaml
k8s/cloud_deps/dev/postgres/postgres_deployment.yaml
k8s/cloud_deps/prod/stan/statefulset_patch.yaml
k8s/cloud_deps/public/postgres/postgres_deployment.yaml
```

File: k8s/cloud/base/indexer_deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ spec:
       containers:
       - name: indexer-server
         image: gcr.io/pixie-oss/pixie-dev/cloud/indexer_server_image
+        imagePullPolicy: IfNotPresent
```
Comment (philkuz) [inline k8s/cloud/base/indexer_deployment.yaml:25]: format looks right to me

**Comment by zasgar** [comment]
ok to test

**Comment by zasgar** [comment]
ok to test

**Comment by philkuz** [comment]
Thanks for making this change! Unfortunately, looks like your editor auto-formatted the code. We don't want to include the formatting with this change.

If you're using VSCode you can disable auto-format for yaml files by adding the following settings change:
```
# cmd+shift+p "Settings (JSON)"
    "[yaml]": { "editor.formatOnSave": false },
```

You'll probably want to revert your last two changes and just redo the ImagePullPolicy for the deployments
```
git reset HEAD~2
```

**Comment by vihangm** [comment]
yaml formatters and editors don't seem to agree on how to format code. however I just created a commit that drops all whitespace changes for you.

Thanks for your contribution, we will try to get this merged in.

**Review by vihangm — APPROVED** [review]

**Review by zasgar — APPROVED** [review]

**Pull request closed without merging**